common/utils/dates.ts

```diff
@@ -10,6 +10,10 @@ import { formatDayDate } from './format-date';
 //      new Date('2022-09-19T00:00:00Z')
 //    );
 //
+// If you're doing local debugging, you can also override the return value
```

Comment:
Nice – i've often wanted to be able to do this

Reply:
I've done it by changing my local clock, and I absolutely do not recommend doing it because a bunch of stuff breaks. 🤒 

Among other things, my browser history is completely ruined. 😂